rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java

```diff
@@ -204,64 +192,54 @@ public static void main(String[] args)
                         getRoom(roomResult.getRoomId());
                         roomIds.add(roomResult.getRoomId());
                         break;
-                    case 2:
-                    {
+                    case 2: {
                         System.out.print("RoomId:");
                         String roomId = br.readLine();
                         updateRoom(roomId);
                         break;
                     }
-                    case 3:
-                    {
+                    case 3: {
                         System.out.print("RoomId:");
                         String roomId = br.readLine();
-                        if(deleteRoom(roomId))
-                        {
+                        if (deleteRoom(roomId)) {
                             roomIds.remove(roomId);
                         }
                         break;
                     }
-                    case 4:
-                    {
+                    case 4: {
                         System.out.print("RoomId:");
                         String roomId = br.readLine();
                         getRoom(roomId);
 
                         break;
                     }
-                    case 5:
-                    {
+                    case 5: {
                         for (String room : roomIds) {
                             System.out.println(room);
                         }
                         break;
                     }
-                    case 6:
-                    {
+                    case 6: {
                         System.out.print("RoomId:");
                         String roomId = br.readLine();
-                        addParticipant(roomId);
+                        addOrUpdateParticipants(roomId);
 
                         break;
                     }
-                    case 7:
-                    {
-                        System.out.print("RoomId:");
-                        String roomId = br.readLine();
-                        updateParticipant(roomId);
+                    case 7: {
+                        System.out.print("Listing all rooms");
```

Comment:
For consistency, either leave this message inside the function itself or here but ensure doing the same for all other functions.

Reply:
Updated